src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts.mdx

```diff
@@ -201,9 +201,9 @@ The following rules apply both to the New Relic user interface and to the REST A
 
     <tr>
       <td>
-        [Custom violation descriptions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions)
+        [Custom incident descriptions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions)
       </td>
-
+        n/a
```

Comment:
🤷🏼‍♂️ FWIW: this was the _only_ cell that did not have _something_ in it for the minimum value, so I added the `n/a` to be consistent with the other items in the table that do not have explicit minimums.